package.json

```diff
@@ -1,56 +1,80 @@
 {
-	"name": "rogue",
-	"version": "0.2.4",
-	"description": "A SSR experience for React that is quick and invisible",
-	"repository": "https://github.com/alidcastano/rogue",
-	"author": "Alid Castano",
-	"license": "MIT",
-	"main": "index.js",
-	"bin": {
-		"rogue": "./bin/rogue"
-	},
-	"engines": {
-		"node": ">=8.0.0",
-		"npm": ">=5.0.0"
-	},
-	"scripts": {
-		"test": "jest"
-	},
-	"peerDependencies": {
-		"react": "^16.4.1",
-		"react-dom": "^16.4.1",
-		"react-router-dom": "^4.3.1"
-	},
-	"dependencies": {
-		"babel-polyfill": "^6.26.0",
-		"connect": "^3.6.6",
-		"express": "^4.16.3",
-		"fs-extra": "^6.0.1",
-		"isomorphic-unfetch": "^2.0.0",
-		"loadable-components": "^2.2.2",
-		"lodash": "^4.17.10",
-		"parcel-bundler": "^1.9.2",
-		"react-dev-utils": "^5.0.1",
-		"react-helmet": "^5.2.0",
-		"react-router": "^4.3.1",
-		"react-tree-walker": "^4.1.0",
-		"serialize-javascript": "^1.5.0"
-	},
-	"devDependencies": {
-		"apollo-boost": "^0.1.10",
-		"babel-jest": "^23.2.0",
-		"babel-preset-env": "^1.7.0",
-		"babel-preset-react": "^6.24.1",
-		"emotion-theming": "^9.2.4",
-		"graphql": "^0.13.2",
-		"graphql-tag": "^2.9.2",
-		"jest": "^23.2.0",
-		"react": "^16.4.1",
-		"react-apollo": "^2.1.8",
-		"react-dom": "^16.4.1",
-		"react-redux": "^5.0.7",
-		"react-router-dom": "^4.3.1",
-		"react-test-renderer": "^16.4.1",
-		"redux": "^4.0.0"
-	}
+  "name": "rogue",
+  "version": "0.2.4",
+  "description": "A SSR experience for React that is quick and invisible",
+  "repository": "https://github.com/alidcastano/rogue",
+  "author": "Alid Castano",
+  "license": "MIT",
+  "main": "index.js",
+  "bin": {
+    "rogue": "bin/rogue.js"
+  },
+  "engines": {
+    "node": ">=8.0.0",
+    "npm": ">=5.0.0"
+  },
+  "scripts": {
+    "lint": "eslint --ext .jsx --ext .js .",
+    "test": "jest"
+  },
+  "husky": {
+    "hooks": {
+      "pre-commit": "lint-staged"
+    }
+  },
+  "lint-staged": {
+    "*.{js,jsx}": [
+      "eslint --fix",
+      "git add"
+    ]
+  },
+  "peerDependencies": {
+    "react": "^16.4.1",
+    "react-dom": "^16.4.1",
+    "react-router-dom": "^4.3.1"
+  },
+  "dependencies": {
+    "babel-polyfill": "^6.26.0",
+    "connect": "^3.6.6",
+    "emotion-theming": "^9.2.4",
+    "express": "^4.16.3",
+    "fs-extra": "^6.0.1",
+    "isomorphic-unfetch": "^2.0.0",
+    "loadable-components": "^2.2.2",
+    "lodash": "^4.17.10",
+    "parcel-bundler": "^1.9.2",
+    "react": "^16.4.1",
+    "react-apollo": "^2.1.8",
+    "react-dev-utils": "^5.0.1",
+    "react-helmet": "^5.2.0",
+    "react-redux": "^5.0.7",
+    "react-router": "^4.3.1",
+    "react-tree-walker": "^4.1.0",
+    "serialize-javascript": "^1.5.0",
+    "styled-components": "^3.3.3"
```

Comment:
some devDependcies changed to dependencies here too

e.g. react-redux and styled-components - how come? 

They're optional for user to use, we only install them so we can run hoc ourselves if we need to for testing

Reply:
Again, these were moved due to lint errors. For example:

`error  'react-redux' should be listed in the project's dependencies, not devDependencies  import/no-extraneous-dependencies`

Reply:
in this case for sure I'd rather ignore linter errors than to add unnecessary dependencies